service/src/test/env_validation.ts

```diff
@@ -0,0 +1,97 @@
+import path from 'path';
+import { fileURLToPath } from 'url';
+import { archLogger } from '../utils/chalk-theme';
+import { TestSuite } from './test_suite';
+import 'dotenv/config';
+
+
+export async function runTests() {
+    archLogger.warn("\n\nVerify service outputs errors for invalid environment variables");
+
+    const cwd = path.dirname(fileURLToPath(import.meta.url));
+    const opts = { sourceFile: '../index.js', timeout: 1000 };
+
+    let testSuite: TestSuite;
+
+    const spawnTestSuite = () => new TestSuite(cwd, process.env.TEST_CONTRACTS_DIRECTORY!);
```

Comment:
wondering if this (except for line 12) could be refactored into a `SetupTestSuite()` method that returns `spawnTestSuite`, since we may need to re-use this code for more tests files in the future.

Reply:
Yeah, good point